templates/compose/linkding.yaml

```
@@ -0,0 +1,20 @@
+# documentation: https://linkding.link/
+# slogan: A self-hosted bookmark manager designed to be minimal, fast, and easy to set up.
+# category: RSS
+# tags: rss, feed
+# logo: svgs/linkding.svg
+# port: 9090
+
+services:
+  linkding:
+    image: sissbruecker/linkding:latest
+    volumes:
+      - './data:/etc/linkding/data:rw'
```

Using a volume mount like suggested below is better because it ensures data is stored persistently and independently of the container lifecycle. This approach also makes it easier to back up or migrate the data, and it avoids potential issues with file permissions that can arise when using bind mounts.

```suggestion
      - 'linkding_data:/etc/linkding/data'
```